tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
this should be scoped to product react. as other packages are not on typescript yet